doc/dev/release.md

```diff
@@ -326,14 +340,6 @@ The release process for the samples repo is simple:
     $ git push --tags
     ```
 
-### (Post-release) Updating the release notes
-
-Add the following line to the top of the GitHub release notes for `v1.3.0`:
```

Comment:
Why is this section being removed?

Reply:
It shows that we should add a reference of the samples in the operator-SDK release notes. Am I right? Have we not one it so far. Should we really do it?

Reply:
We did not it so far. I removed by mistake. 
Added back.